flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -76,17 +75,21 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
   final RouteNameExtractor? _routeNameExtractor;
   final AdditionalInfoExtractor? _additionalInfoProvider;
 
+  ISentrySpan? _initialTransaction;
```

Comment:
Is this going to stay in memory forever? or does it set to null after finished and sent?
Otherwise, we could set it to null ourselves once we know it's done.

Reply:
It's unused, will remove.